build/agent/agent.yaml

```diff
@@ -0,0 +1,32 @@
+# This manifest is for testing purposes and is not the final manifest for the agent.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-agent
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx-agent
+  template:
+    metadata:
+      labels:
+        app: nginx-agent
+    spec:
+      serviceAccountName: default
```

Comment:
shall we also disable auto mounting of credentials?

Reply:
I'm assuming you mean `automountServiceAccountToken: false`?